3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/model/ShoppingCart.java

```diff
@@ -0,0 +1,16 @@
+package com._6core.platform.shopping.cart.app.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+  @Id private String id;
+  private String[] productIds;
```

Comment:
Did you test that this mapping works? I have some doubts. 
```suggestion
  private String[] productIds;
```

Reply:
It works, I tried it on R2dbcRepository. I also tried the option with “private List<String> productIds;”, everything also works, but I decided to leave the original option.